sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ErrorParserSuite.scala

```diff
@@ -63,4 +66,10 @@ class ErrorParserSuite extends SparkFunSuite {
       "Combination of ORDER BY/SORT BY/DISTRIBUTE BY/CLUSTER BY is not supported",
       "^^^")
   }
+
+  test("SPARK-27901: misleading error message due to problematic antlr grammar") {
```

Comment:
Let's use `SPARK-21136` for this PR. `SPARK-27901` is an umbrella issue which is not proper in this context.